S21/rebelDi/tests/diana/WordToLowerCaseTest.java

```diff
@@ -0,0 +1,72 @@
+package diana;
+
+import org.junit.Test;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+import static org.junit.Assert.*;
+
+public class WordToLowerCaseTest {
+
+    /**
+     * This is the test method for WordToLowerCase class,
+     * that tests the result of changeWordToLowerCase()
+     */
+    @Test
+    public void changeWordToLowerCase() throws IOException {
+        List<String[]> wordsToProcess = openTSV();
+        for(String[] wordData:wordsToProcess){
+            String wordForLowerCase = wordData[0];
+            String language = wordData[1];
+            String correctWordInLowerCase = wordData[2];
+
+            System.out.println("Word for lowercase: " + wordForLowerCase);
+
+            String wordInLowerCase = WordToLowerCase.changeWordToLowerCase(wordForLowerCase, language);
+            assertEquals("The word was processed correctly", correctWordInLowerCase, wordInLowerCase);
+        }
+    }
+
+    /**
+     * This is the function for opening the tsv file with examples of words to convert
+     *
+     * @return all the data from file
+     */
+    public static List<String[]> openTSV () throws IOException {
+        List<String[]> listOfDataFromFile = new ArrayList<>();
+
+        StringTokenizer stringTokenizer; // used to get data from row for each column
```

Comment:
I would like to see how this tokenizer works.

Reply:
It just get the line from the file, looks for the \t as a stop point and divides the line by words